lib/mondrian_redis_segment_cache/cache.rb

```diff
@@ -93,7 +94,6 @@ def put(segment_header, segment_body)
         end
       end
 
-      publish_created_to_listeners(header_base64)
```

Comment:
so we are now going to rely on `reconcile_local_set_with_redis` to always do the publishing?